catalog/app/containers/Auth/SSOAzure.js

```diff
@@ -1,4 +1,4 @@
-import { push } from 'connected-react-router/esm/immutable'
+import { push } from 'connected-react-router'
```

Comment:
please backlog ditching `connected-react-router` as well

Reply:
I don't need task for this